pyramid/file.go

```diff
@@ -0,0 +1,37 @@
+package pyramid
+
+import "os"
+
+type File struct {
+	fh      *os.File
```

Comment:
Can this be a ReadWriteSeeker or somesuch?  Not everything is a file.  (E.g. for testing a pure-memory fake file might be really useful).

Reply:
Not really, `pebble` sstable reader requires the Entire File interface. see https://github.com/cockroachdb/pebble/blob/e6d62a79c8e3f3bec3cc1f49c93ad4a311985d40/sstable/reader.go#L2200